chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -812,6 +769,10 @@
                     <key>authorities</key>
                     <value>ROLE_ADMIN;ROLE_ONC;ROLE_ONC_STAFF</value>
                 </entry>
+                <entry>
```

Comment:
This job does not get "modified" to have tihe acbSpecific proprety when Tomcat starts. I think you need to have a section at the top that deletes the job and then leave this code here so that when Tomcat starts it will re-add the job with the correct properties. Note that this will delete all user triggers for the job and either the users or we will have to re-add them when this ticket is released. Will need a note in the Wiki for that.